### ISSUE TITLE
[Feature Request] Settings to change the color of @started/cancelled/done tags

### ISSUE THREAD

**Issue by Zooce** [fabiospampinato/vscode-todo-plus#33]
When you have a lot of tasks marked as either `@started` or `@cancelled` or `@done` it becomes rather difficult to see which ones you're actively working on because all of them are the same color (it's a yellow-ish color). It would be really nice to have the option to change the colors of those tags specifically.

**Comment by fabiospampinato** [comment]
You can already do this my making them special tags, just add this to your settings (change the colors to your likings):

```json
  "todo.tags.names": ["critical", "high", "low", "today", "done", "cancelled", "started"], // Special tags' names
  "todo.tags.backgroundColors": ["#e54545", "#e59f45", "#e5d145", "#ae81ff", "#a6e22e", "#f92672", "#ff0000"], // Special tags' background colors
  "todo.tags.foregroundColors": ["#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000"], // Special tags' foreground colors
```

This is the result:

<img width="831" alt="screen shot 2018-02-27 at 13 25 07" src="https://user-images.githubusercontent.com/1812093/36728651-c9088852-1bc1-11e8-9ee3-d326cfc8d313.png">

Currently the text inside the parentheses doesn't get colorized though, I'll push an update fixing this!

**Comment by fabiospampinato** [comment]
> Currently the text inside the parentheses doesn't get colorized though, I'll push an update fixing this!

Fixed in v1.8.1.
